cunumeric/_ufunc/ufunc.py

```diff
@@ -55,6 +55,44 @@
 Multiple GPUs, Multiple CPUs
 """
 
+_MULTIOUT_UNARY_DOCSTRING_TEMPLATE = """{}
+
+Parameters
+----------
+x : array_like
+    Only integer and boolean types are handled.
+out : tuple[ndarray or None], or None, optional
```

Comment:
Are we handling the `out1` and `out2` arguments?

Reply:
we aren't, will change it today.

Reply:
should be fixed now